modules/admin/ref-rbac-rest-api-endpoints.adoc

```diff
@@ -834,6 +918,18 @@ Updates a condition policy for a specified ID.
 |Required
 |===
 
+|`name`
+|Name of the role
+|String
+|Required
+|===
+
+|`metadata.description`
+|The description of the role
+|String
+|Optional
+|===
+
```

Comment:
Could you please check the formatting of the table? It does not look correctly formatted.